src/main/java/io/cryostat/net/web/http/api/v1/RecordingsPostHandler.java

```diff
@@ -148,7 +148,7 @@ public String path() {
 
     @Override
     public boolean isAsync() {
-        return false;
+        return true;
```

Comment:
This handler is `async` since `validateRecording` runs on a worker thread using `vertx.executeBlocking`, correct?

Reply:
Yea exactly, all of the substantial work performed in his handler is now parcelled off with `executeBlocking`, so the from the main event loop thread's perspective it should be async.